examples/meta/src/kernel/anova.sg

```diff
@@ -0,0 +1,20 @@
+File traindat = csv_file("../../data/fm_train_real.dat")
+File testdat = csv_file("../../data/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Labels feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel anova_kernel = kernel("ANOVAKernel")
```

Comment:
could you use the new kwargs style here?
`Kernel k = kernel("ANOVEKernel", cardinality=2)`